dev_tools/scripts/db_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+CONFIGFILE="dev_tools/config.yaml" go run . --action pg_dump
+
+docker exec db psql -U postgres -d postgres -c "DROP DATABASE test;"
+
+docker exec db psql -U postgres -d postgres -c "CREATE DATABASE test;"
+
+DUMPFILE=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".sqldump" | cut -d '/' -f4)
+echo "restoring databse from file $DUMPFILE"
+CONFIGFILE="dev_tools/config.yaml" go run . --action pg_restore --name "$DUMPFILE"
```

Comment:
Why is `CONFIGFILE` all over the place?

Reply:
Because we need to tell the app what configurations to use.

Reply:
Why not just `export CONFIGFILE="dev_tools/config.yaml"`

And later running the code?